### DISCUSSION TITLE
How to inject

### DISCUSSION THREAD

**Discussion by Rutherther** [Rutherther/NosSmooth#67] [Announcements]
@plsfixrito Hello as it looks like you are interested in NosSmooth, I have written a short wiki page on how to get started with injecting NosSmooth into NosTale process if that was not clear. In case you had any problems with that, feel free to ask any questions.

https://github.com/Rutherther/NosSmooth.Local/wiki/How-to-inject-NosSmooth-to-NosTale

**Comment by plsfixrito** [comment]
Hello, thank you for the mention. I have seen the library and already started testing with injection and using functions from the project. I will contribute when I have additions to the project.
